File: code_corpus/manual/sort_2.m
function dataOut = insertionSort(data)
  len = size(data,2);
  for j = 2:len;
    key = data(j);
    i = j-1;
    while(i>0 && data(i)>key)
      data(i+1) = data(i);
      i = i-1;
      data(i+1)=key;
    end
  end
dataOut = data;

File: code_corpus/manual/sort_2.java
public int[] InsertionSort(int[] data){
  int len = data.length;
  int key = 0;
  int i = 0;
  for(int j = 1;j<len;j++){
    key = data[j];
    i = j-1;
    while(i>=0 && data[i]>key){
      data[i+1] = data[i];
      i = i-1;
      data[i+1]=key;
    }
  }
  return data;
}
